### PR TITLE
license: improvements on license

### PR DESCRIPTION
**Pull request by liujing2**
The crate needs Apache-2.0 or BSD-3-Clause but some files are not correctly
described with the license. Make all files correct on license declaration.

### CODE REVIEW

File: src/signal.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Copyright 2019 Intel Corporation. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
 //
 // Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
```
Comment (jiangliu) [inline src/signal.rs]: This line should be removed too.

Comment (liujing2) [reply src/signal.rs]: Line 5 is copied from Amazon license together with line 4. Do you mean we only tell the companies, removing the original license that companies used, and leave one line for declaring final license?

Comment (jiangliu) [reply src/signal.rs]: not sure whether it makes sense to include multiple "SPDX-License-Identifier" header:)

Comment (liujing2) [reply src/signal.rs]: @andreeaflorescu @sameo Do you know the good way to list the companies and licenses that might be different ?

Comment (andreeaflorescu) [reply src/signal.rs]: @liujing2 I really don't know what is the best approach here. I think it makes sense to send an email to the rust-vmm email list and ask there.

Comment (sameo) [reply src/signal.rs]: The goal of the SPDX id is to provide a single, simple and unified way to expose under which terms a source code file is licensed. Havinf multiple `SPDX-License-Identifier` is confusing and goes against the point of using SPDX ids. So I'd go with a single one.

In that case, Amazon licensed their piece of code under the Apache 2.0 license (exclusively, not dual licensed). So unless Amazon wants to re license (for dual licensing), I think the single SPDX identifier that we should use here is:

```
SPDX-License-Identifier: Apache-2.0
```

Comment (sameo) [reply src/signal.rs]: If the piece of code also includes some BSD-3-Clause code from crosvm, then we should use:

```
SPDX-License-Identifier: (Apache-2.0 AND BSD-3-Clause)
```

Comment (liujing2) [reply src/signal.rs]: Thanks for your guidance.  The PR is updated. :)

File: src/eventfd.rs [hunk 2]
```diff
@@ -1,9 +1,8 @@
 // Copyright 2019 Intel Corporation. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
 //
 // Copyright 2017 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
```
Comment (sameo) [inline src/eventfd.rs]: This is APACHE AND BSD, as I assumed Intel added Apache licensed code.

File: src/fallocate.rs [hunk 3]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+//
+// SPDX-License-Identifier: BSD-3-Clause
```
Comment (sameo) [inline src/fallocate.rs]: Ditto.

File: src/file_traits.rs [hunk 4]
```diff
@@ -1,6 +1,6 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
 // Copyright 2018 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE-BSD-3-Clause file.
 //
 // SPDX-License-Identifier: BSD-3-Clause
```
Comment (sameo) [inline src/file_traits.rs]: Ditto.

File: src/seek_hole.rs [hunk 5]
```diff
@@ -1,6 +1,6 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
 // Copyright 2018 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE-BSD-3-Clause file.
 //
 // SPDX-License-Identifier: BSD-3-Clause
```
Comment (sameo) [inline src/seek_hole.rs]: Ditto.

File: src/syslog.rs [hunk 6]
```diff
@@ -1,9 +1,8 @@
 // Copyright 2019 Intel Corporation. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
 //
 // Copyright 2017 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
```
Comment (sameo) [inline src/syslog.rs]: APACHE AND BSD

File: src/tempdir.rs [hunk 7]
```diff
@@ -1,6 +1,6 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
 // Copyright 2017 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE-BSD-3-Clause file.
 //
 // SPDX-License-Identifier: BSD-3-Clause
```
Comment (sameo) [inline src/tempdir.rs]: Ditto.

**Pull request merged**